src/incubator-tooltip.html

```diff
@@ -163,17 +176,30 @@
             this._targetElement.removeEventListener('blur', this._boundHide);
             this._targetElement.removeEventListener('tap', this._boundHide);
           }
+
+          if (!this.manual && !this._moved) {
+            this.removeEventListener('mouseenter', this._boundShow);
+            this.removeEventListener('mouseleave', this._boundHide);
+          }
         }
 
         _setPosition() {
           if (!this.isAttached) {
             return;
           }
-          const parentRect = this.offsetParent.getBoundingClientRect();
+          let parentRectTop = 0,
```

Comment:
What is the point of making the logic more complicated?

Reply:
if it is absolutely necessary pls comment that.

Reply:
offsetParent is null for empty tags.